src/browser-installer/install.ts

```diff
@@ -0,0 +1,112 @@
+import _ from "lodash";
+
+/**
+ * @returns path to browser binary
+ */
+export const installBrowser = async (
+    browserName?: string,
+    browserVersion?: string,
+    { force = false, installWebDriver = false } = {},
+): Promise<string | null> => {
+    const unsupportedBrowserError = new Error(
+        [
+            `Couldn't install browser '${browserName}', as it is not supported`,
+            `Currently supported for installation browsers: 'chrome', 'firefox`,
+        ].join("\n"),
+    );
+
+    if (!browserName) {
+        throw unsupportedBrowserError;
+    }
+
+    if (!browserVersion) {
+        throw new Error(
+            `Couldn't install browser '${browserName}' because it has invalid version: '${browserVersion}'`,
+        );
+    }
+
+    if (/chrome/i.test(browserName)) {
+        const { installChrome, installChromeDriver } = await import("./chrome");
+
+        return installWebDriver
+            ? await Promise.all([
+                  installChrome(browserVersion, { force }),
+                  installChromeDriver(browserVersion, { force }),
+              ]).then(binaries => binaries[0])
+            : installChrome(browserVersion, { force });
+    } else if (/firefox/i.test(browserName)) {
+        const { installFirefox, installLatestGeckoDriver } = await import("./firefox");
+
+        return installWebDriver
+            ? await Promise.all([
+                  installFirefox(browserVersion, { force }),
+                  installLatestGeckoDriver(browserVersion, { force }),
+              ]).then(binaries => binaries[0])
+            : installFirefox(browserVersion, { force });
+    } else if (/edge/i.test(browserName)) {
+        const { installEdgeDriver } = await import("./edge");
+
+        if (installWebDriver) {
+            await installEdgeDriver(browserVersion, { force });
+        }
+
+        return null;
```

Comment:
Because we can't download `edge` or `safari`. We return `null` so `NewBrowser` wouldn't set desiredCapabilities.
With `safari` and `edge` currently, as we are using globally installed browser version, we don't need to specify binaryPath. This will change after edge autodownload support though.